CRM/Core/Payment/Manual.php

```diff
@@ -18,6 +18,22 @@ class CRM_Core_Payment_Manual extends CRM_Core_Payment {
 
   protected $result;
 
+  /**
+   * This support variable is used to allow the capabilities supported by the Dummy processor to be set from outside the class.
+   * Initially these capabilities are set to TRUE, however they can be altered by calling the setSupports function directly from outside the class.
```

Comment:
If you're updating this anyway can we add some more explanation, and it's not the Dummy processor. Maybe something like

This support variable is used to allow the capabilities supported by the Manual processor to be set from unit tests so that we don't need to create a lot of new processors to test combinations of features.